virttest/remote.py

```diff
@@ -1442,3 +1455,31 @@ def run_command(self, vm_ip, command, vm_user="root", runner=None,
             raise exceptions.TestFail("Failed to run '%s' in the VM: %s",
                                       cmd, detail)
         return ret
+
+    def cmd_output_safe(self, cmd, timeout=CMD_TIMEOUT):
+        """
+        Unify the interface for session.cmd_output_safe()
+        :param cmd: The command to execute
+        :param timeout: Ignored so far
+        :return: cmd_result.stdout_text
+        """
+        return self.run_command(cmd).stdout_text.strip()
+
+    def cmd_status(self, cmd, safe=True):
+        """
+        Unify the interface for session.cmd_status()
```

Comment:
1 blank line needed between description and params

Reply:
update